docker/rucio_client/scripts/updateDDMQuota

```diff
@@ -2,54 +2,36 @@
 
 import math
 from rucio.client import Client
+from tabulate import tabulate
 client = Client()
 
-# ddm_quota are weight given to RSEs based on the amount of free space
-# This is calulated as static use - rucio use
-# The rule evaluation algorithm uses a weighted random selection of RSEs based on this value
-
-# NOTE: This probably needs to be reviewed after an assement of age of dynamic data at different sites
-# and if this can be used to normalise that
-
+# See LINK for documentation and please update it if you change the logic
```

Comment:
You meant to add links to docs here, I guess?

Reply:
yes, but the MR needed to be merged for now. Anyway, I merged it and I'm adding it here.